akka-distributed-cluster-docs/src/main/paradox/guide/2-service-to-service.md

```diff
@@ -14,8 +14,6 @@ a downstream analytics service.
 
 ## gRPC transport for consuming events
 
-FIXME graphic that is more overview and less step by step? (this is the same as projection gRPC)
```

Comment:
can't we use this this image? I made a more overview in feature summary, but maybe good to go one step deeper here?

Reply:
Yes, maybe it is fine.